sources/zafira-services/src/main/java/com/qaprosoft/zafira/services/services/application/FilterService.java

```diff
@@ -104,4 +109,17 @@ public Subject getStoredSubject(Subject.Name name) {
     public String getTemplate(FilterType filter, Template template) {
         return freemarkerUtil.getFreeMarkerTemplateContent(template.getPath(), filter);
     }
+
+    public boolean isFilterExists(Filter filter) {
```

Comment:
Can you select in one db call? Something like `repo.existsByNameAndPublicAccess` etc? Maybe i am missing something here, correct me if i'm wrong

Reply:
@nikitasidorevich I was thought about sql-s but filters count is not grand and it is not possible to implement bot ugly query to predefine these predicates